tests/unit/setup-context-test.js

```diff
@@ -174,6 +176,351 @@ module('setupContext', function (hooks) {
         assert.equal(bar.get('someProp'), 'derp', 'property updated');
       });
 
+      module('deprecations', function () {
```

Comment:
Should we have a test that assert.throws if `getContext` return `undefined`?

Reply:
Oops, your right i forget to test `getDeprecationsForContext`